src/test/java/org/jenkinsci/plugins/multiplescms/BuilderTest.java

```diff
@@ -0,0 +1,24 @@
+package org.jenkinsci.plugins.multiplescms;
+
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.Label;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class BuilderTest {
+
+    @Rule
+    public JenkinsRule jenkins = new JenkinsRule();
+
+    final String name = "Bobby";
+
+    @Test
+    public void testConfigRoundtrip() throws Exception {

```

Comment:
this doesn't appear to do anything?

Reply:
I believe this satisfies a minimal formality level to get the plugin built - that there *is* some test to run and report to inspect.

Reply:
That said, I would welcome actual tests appearing eventually... though don't hold my breath for them with the plugin's outlook gloom as people migrate from legacy (freestyle etc.) jobs where this plugin really helps, into pipelines that can just use many checkout steps.